src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
Can new transactions arrive every 300ms? That will saturate cpu easily.

What we really need - is to make sure that there is some pause between `computeAutobridgedOffersWrapper` calls, so cpu can process something else. Maybe we should do that check?